src/SnackbarProvider/SnackbarProvider.tsx

```diff
@@ -306,7 +308,18 @@ class SnackbarProvider extends Component<SnackbarProviderProps, State> {
 
         const snackbars = Object.keys(categ).map((origin) => {
             const snacks = categ[origin];
-            const [nomineeSnack] = snacks;
+            const [nomineeSnack] = snacks.sort((a, b) => a.displayOrder - b.displayOrder);
```

Comment:
It might be easier to understand show the larger ones from top to bottom.
Please add which is which to [comment](https://github.com/oviceinc/notistack/pull/1/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR307).
```suggestion
            const [nomineeSnack] = snacks.sort((a, b) => b.displayOrder - a.displayOrder);
```